app/controllers/api/v1/sessions_controller.rb

```diff
@@ -35,13 +36,18 @@ def destroy
       private
 
       def session_params
-        params.require(:session).permit(:email, :password)
+        params.require(:session).permit(:email, :password, :remember_me)
       end
 
       # Signs In the user
       def sign_in(user)
         session[:user_id] = user.id
       end
+
+      def remember(user)
+        cookie.permanent.signed[:user_id] = user.id
+        cookie.permanent[:remember_token] = user.generate_remember_token!
```

Comment:
Besides that I don't think that this the simplest way of achieving this.
But even with the current solution the `remember_token` is **NOT** tamper proof and require **at least** cryptographic signature.